js/bitmex.js

```diff
@@ -323,14 +325,11 @@ module.exports = class bitmex extends Exchange {
         }, params);
         let bid = undefined;
         let ask = undefined;
-        if (this.options['fetchTickerQuotes']) {
-            let quotes = await this.publicGetQuoteBucketed (request);
-            let quotesLength = quotes.length;
-            let quote = quotes[quotesLength - 1];
-            bid = this.safeFloat (quote, 'bidPrice');
-            ask = this.safeFloat (quote, 'askPrice');
-        }
-        let tickers = await this.publicGetTradeBucketed (request);
+        let instruments = await this.publicGetInstrument (request1);
+        let instrument = instruments[0];
+        bid = this.safeFloat (instrument, 'bidPrice');
+        ask = this.safeFloat (instrument, 'askPrice');
+        let tickers = await this.publicGetTradeBucketed (request2);
```

Comment:
I will rewrite this a bit later so that it will send just one request.
The `instrument` endpoint indeed returns all the data, so we don't need `publicGetTradeBucketed` here at all.

Reply:
Well, okay, thanks. Did not get it by myself

Reply:
@hroff-1902 no worries, appreciate your effort! Will merge it soon! Thx!

Reply:
BTW, in the reference _official_ bitmex client they fetch ticker data via WS api, not REST...